tests/UI/pages/FO/FObasePage.js

```diff
@@ -125,6 +125,16 @@ class FOBasePage extends CommonPage {
     return this.clickAndWaitForNavigation(page, selector);
   }
 
+  /**
+   * Click on bread crumb link
+   * @param page {Page} Browser tab
+   * @param link {string} Link to click on
+   * @returns {Promise<void>}
+   */
+  async clickOnBreadCrumbLink(page, link) {
+    await this.waitForSelectorAndClick(page, `#wrapper nav.breadcrumb a[href*=${link}]`);
```

Comment:
Some suggestions : 
- Declare the selector in selectors section
- Use `clickAndWaitForNavigation` to avoid navigation problems